### PR TITLE
fix: do not return empty JSON object if no content

### LINKED ISSUE #17: Do not return empty JSON objects
The API should return no content, if there is none, not an empty JSON object.

https://github.com/bifravst/sim-details/blob/b2b9e635711a3520bb63d202c6c2a46cdcc4b2b1/lambda/getBasicInformationLambda.ts#L48

### PR DESCRIPTION
**Pull request by Lenakh97**
Closes #17

### CODE REVIEW

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Add a test that ensures there is no data in the body (content-lenght is 0 and body is empty)

File: e2e-tests/e2eTest.spec.ts [hunk 1]
```diff
@@ -30,15 +30,21 @@ void describe('e2e-tests', () => {
 	})
 	void it('should return statusCode 409 and cache max-age=60 when the SIM information is not in DB', async () => {
 		const req = await fetchData(getRandomICCID())
+		const text = await req.text()
 		const expectedCacheControl = 'public, max-age=60'
 		assert.equal(req.headers.get('cache-control'), expectedCacheControl)
 		assert.equal(req.status, 409)
+		assert.equal(req.headers.get('content-length'), '0')
+		assert.equal(text, '')
```
Comment (coderbyheart) [inline e2e-tests/e2eTest.spec.ts]:
```suggestion
		const expectedCacheControl = 'public, max-age=60'
		assert.equal(req.headers.get('cache-control'), expectedCacheControl)
		assert.equal(req.status, 409)
		assert.equal(req.headers.get('content-length'), '0')
		assert.equal(await req.text(), '')
```

style: move variable declaration close to where it is used. Since variable is not re-used, this can be done directly.

**Review by coderbyheart — APPROVED** [review]
